assets/src/edit-story/components/canvas/carousel/index.js

```diff
@@ -309,16 +325,25 @@ function Carousel() {
           </MenuIconsWrapper>
         </MenuArea>
       </Wrapper>
-      {isGridViewOpen && (
-        <Modal
-          isOpen={isGridViewOpen}
-          onRequestClose={closeModal}
-          contentLabel={__('Grid View', 'web-stories')}
-          closeButtonLabel={__('Back', 'web-stories')}
-        >
+      <Modal
+        open={isGridViewOpen}
+        onClose={closeModal}
+        contentLabel={__('Grid View', 'web-stories')}
+        overlayStyles={{
+          alignItems: 'flex-start',
+        }}
+        contentStyles={{
+          pointerEvents: 'none',
+          flex: 1,
+        }}
+      >
+        <GridViewContainer>
+          <TextButtonStyled onClick={() => closeModal()}>
+            {__('Back', 'web-stories')}
```

Comment:
This button in the grid view modal looks now different from before. Expected / OK?

Also, the slider in this modal is now below the button, not on the same level. Expected / OK?

Reply:
Doesn't look like the design for the grid view modal has changed.

Also, perhaps not a significant difference but the text for the link modal is "OK" on the design and not "Back" (that's for Grid View Modal).

Reply:
Signed off with @samitron7